terraform.sample.yml

```diff
@@ -25,10 +25,6 @@
     # this variable.
     # consul_acl_datacenter: your_primary_datacenter
 
-    # consul_dns_domain is repeated across these plays so that all the
-    # components know what information to use for this values to help with
-    # service discovery.
-    consul_dns_domain: consul
```

Comment:
we should remove the re-definition of `consul_dns_domain` from `addons/glusterfs.yml` and `kubernetes.yml` too